version: "3.8"

services:
  django-app:
    build:
      context: .
    volumes:
      - .:/corruption
    ports:
      - "8000:8000"
    depends_on:
      - postgres-db
    environment:
      - DEBUG=1
      - DJANGO_SETTINGS_MODULE=corruption.settings
      - DATABASE_HOST=postgres-db
      - DATABASE_PORT=5432
      - DATABASE_NAME=corruption_db
      - DATABASE_USER=corruption_db_user
      - DATABASE_PASSWORD=Cr6PzluGY8XQhQGygYz031rKqYRCoCvD

  postgres-db:
    image: postgres:13
    volumes:
      - postgres_data:/var/lib/postgresql/data
    ports:
      - "5432:5432"
    environment:
      POSTGRES_DB: corruption_db
      POSTGRES_USER: corruption_db_user
      POSTGRES_PASSWORD: Cr6PzluGY8XQhQGygYz031rKqYRCoCvD

volumes:
  postgres_data:
